call out support for AlloyDB via dbt-postgres (#1926)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

### Connecting via an SSH Tunnel

> To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, check the "Use SSH Tunnel" option when creating your connection. When configuring the tunnel, you'll need to supply the hostname, username, and port for the bastion server.

Reviewer: ```suggestion
To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, select the **Use SSH Tunnel** option when creating your connection. When configuring the tunnel, you must supply the hostname, username, and port for the bastion server.
```

Reviewer: the docs team is working on a style guide, which we've started to use (and will publish soon!). for checkboxes, "select" and "choose" are the preferred words to use: https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md#checkboxes
